earth2studio/lexicon/hrrr.py

```diff
@@ -187,8 +188,8 @@ def build_vocab() -> dict[str, str]:
                 prs_variables[f"{id}{level:d}"] = f"sfc::fcst::{level} mb::{variable}"
 
         hybrid_levels = list(range(1, 51))
-        hybrid_names = ["UGRD", "VGRD", "HGT", "TMP", "SPFH", "PRES"]
-        e2s_id = ["u", "v", "z", "t", "q", "p"]
+        hybrid_names = ["UGRD", "VGRD", "HGT", "TMP", "SPFH", "PRES", "HGT"]
+        e2s_id = ["u", "v", "z", "t", "q", "p", "Z"]
```

Comment:
It's there, right? Both `z` and `Z` are in the list

Reply:
Whats capital Z? is it for just hgt?

Reply:
If its geo potential height maybe it should be `gh` as the ID? we try to make to kinda consistent with ECMWF within reason.
https://codes.ecmwf.int/grib/param-db/?search=geo

Reply:
Yeah, geopotential height. I'm not opposed to swapping to `gh` instead of `Z` but it'll require updating the NGC package as well. @dallasfoster what do you think?

Reply:
Let's keep `Z` for now. I am not convinced  `gh` is superior.